File: frontend/src/constants/events.ts
export const EVENT_NAMES = {
  THEME_CHANGED: 'theme-changed',
  CAN_WORK_CHANGED: 'can-work-changed',
  STICKY_SIDE_CHANGED: 'sticky-side-changed',
  MOUSE_ENTER_FLOATING: 'mouse-enter-floating',
  MOUSE_LEAVE_FLOATING: 'mouse-leave-floating',

  MESSAGE_RESPONSED: "message-responsed",
  TASK_STATUS_CHANGED: "task-status-changed",
  OPERATE_FAILED: "operate-failed",

  NOTIFY: "notify",

  // 任务执行相关事件
  TASK_EXECUTION_STARTED: "task-execution-started",
  TASK_EXECUTION_COMPLETED: "task-execution-completed",
} as const

File: background/constant/events.go
package constant

const (
	EventThemeChanged       = "theme-changed"
	EventCanWorkChanged     = "can-work-changed"
	EventStickySideChanged  = "sticky-side-changed"
	EventMouseEnterFloating = "mouse-enter-floating"
	EventMouseLeaveFloating = "mouse-leave-floating"

	EventMessageResponsed  = "message-responsed"
	EventTaskStatusChanged = "task-status-changed"
	EventOperateFailed     = "operate-failed" // model.Step

	EventNotify = "notify"

	// 任务执行相关事件
	EventTaskExecutionStarted   = "task-execution-started"
	EventTaskExecutionCompleted = "task-execution-completed"
)
